src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -506,6 +506,7 @@ def load_arguments(self, _):
     with self.argument_context('vmss extension set', min_api='2017-12-01') as c:
         c.argument('force_update', action='store_true', help='force to update even if the extension configuration has not changed.')
         c.argument('extension_instance_name', extension_instance_name_type)
+        c.argument('provision_after_extensions', options_list=['--provision-after-extensions'], nargs='+', help='Space-separated list of extension names after which this extension should be provisioned. These extensions must already be set on the vm.')
```

Comment:
`options_list=['--provision-after-extensions']` is redundant?

Reply:
Yes it is @yugangw-msft. Will remove.